virtme/guest/virtme-init

```diff
@@ -4,8 +4,11 @@
 # Licensed under the GPLv2, which is available in the virtme distribution
 # as a file called LICENSE with SHA-256 hash:
 # 8177f97513213526df2cf6184d8ff986c675afb514d4e68a404010521b880643
+set -u
```

Comment:
I know it makes debugging easier for us, but it can also make the script more "faulty", I'd drop this.

Reply:
Ok.